packages/ui/components/apps/AppCard.tsx

```diff
@@ -200,7 +205,9 @@ const InstallAppButtonChild = ({
 
   if (
     !userAdminTeams?.length ||
-    appCategories.some((category) => category === "calendar" || category === "conferencing")
```

Comment:
Can we extract it out as `doesAppSupportTeamInstall()` to be used across components.